test/plain-list/AutosuggestApp.test.js

```diff
@@ -578,13 +578,25 @@ describe('Default Autosuggest', () => {
       shouldRenderSuggestions.resetHistory();
     });
 
-    it('should be called with the right parameters', () => {
+    it('should be called with the right parameters during input', () => {
       focusAndSetInputValue('e');
-      expect(shouldRenderSuggestions.callCount).to.equal(4);
+      blurInput();
+      expect(shouldRenderSuggestions.callCount).to.equal(5);
       expect(shouldRenderSuggestions.getCall(0).args).to.deep.equal(['', 'input-focused']);
       expect(shouldRenderSuggestions.getCall(1).args).to.deep.equal(['e', 'input-changed']);
       expect(shouldRenderSuggestions.getCall(2).args).to.deep.equal(['e', 'suggestions-updated']);
       expect(shouldRenderSuggestions.getCall(3).args).to.deep.equal(['e', 'render']);
+      expect(shouldRenderSuggestions.getCall(4).args).to.deep.equal(['e', 'input-blurred']);
+    });
+
+    it('should be called with the right parameters when revealing/hiding suggestions', () => {
+      focusInput();
+      clickUp();
+      clickEscape();
+      expect(shouldRenderSuggestions.callCount).to.equal(3);
+      expect(shouldRenderSuggestions.getCall(0).args).to.deep.equal(['', 'input-focused']);
+      expect(shouldRenderSuggestions.getCall(1).args).to.deep.equal(['', 'suggestions-revealed']);
+      expect(shouldRenderSuggestions.getCall(2).args).to.deep.equal(['', 'escape-pressed']);
     });
 
```

Comment:
Do you want to add a test for the use case you describe in the bug? If I understand your use case correctly, `shouldRenderSuggestions('', 'arrow-pressed')` returning true will force all selections to be shown?

Reply:
Bit fiddly, but I've found a nice way with `sinon.stub` - now done.

Reply:
Nice!